util/dataset.go

```diff
@@ -24,6 +24,13 @@ const (
 	DirIngestPipeline = "ingest-pipeline"
 )
 
+var ValidTypes = map[string]interface{}{
+	"logs":    nil,
```

Comment:
This might cause problems in the future if someone would like to check if the value `logs` exists. They would get a `nil` value :) Maybe bool instead?

Reply:
I went an other approach and made the second part a String which is something we could also show in a UI if needed. Thinking of the case that someday we might have a `/types` endpoint if someone wants to know all the available dataset types.